app/views/root/gem_layout.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render partial: "gem_base", locals: {
+  full_width: false,
```

Comment:
I guess we can omit this here, as it defaults to false in layout_header, and if nil is passed in `gem_base` it will still be false

Reply:
Omitting it throws an error, since `full_width` isn't declared in gem_base - to get around this I've set `full_width` to default to false so it doesn't need to be set.

Reply:
ah, right – these are the partial locals, not the component call – my bad